microsoft/botframework-solutions
# [Enterprise Bot Generator][TypeScript] Implement Mocha as Test Framework

## Description

- Delete Jest.
- Implement Mocha.

## Related Issue
Fix [#613](https://github.com/Microsoft/AI/issues/613) 

## Testing Steps

1. Open `AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise`.
2. Run `npm i` and check that **Mocha** dependency was installed

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
